content/PIV/Guides/Android_code_signing.adoc

```diff
@@ -4,8 +4,18 @@ key and certificate.
 
 === Prerequisites
 * a YubiKey with the PIV application
-* the yubico-piv-tool software
-* the OpenSC software
+* the yubico-piv-tool software, available link:https://developers.yubico.com/yubico-piv-tool/Releases/[here] or on link:https://github.com/Yubico/yubico-piv-tool/releases[GitHub] or
```

Comment:
"available on link:..." maybe?

Reply:
This is how it looks when rendered, I think it makes sense:

![image](https://user-images.githubusercontent.com/10159615/65587500-3dfb4580-df86-11e9-9bf4-69831d1f589d.png)

What do you think?